From ccf66921f87d2a9408339704249a60a3e10ee91c Mon Sep 17 00:00:00 2001
From: Holger Hans Peter Freyther <zecke@selfish.org>
Date: Thu, 19 Mar 2009 14:34:06 +0800
Subject: [PATCH] Disable hide/show buttons for WebKit based browsers for now

For content rendering we will need another way to encode
these sections. A prototype should be done with JavaScript...
---
 app/models/parsers/xhtml.rb |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/app/models/parsers/xhtml.rb b/app/models/parsers/xhtml.rb
index 411079a..20f5ef7 100644
--- a/app/models/parsers/xhtml.rb
+++ b/app/models/parsers/xhtml.rb
@@ -98,8 +98,8 @@ module Parsers
         headings += 1 
 
         # generate the HTML we are going to inject
-        buttons = "<button class='section_heading show' section_id='#{headings}'>Show</button><button class='section_heading hide' style='display: none' section_id='#{headings}'>Hide</button>"
-        base = "<h2#{$1}#{buttons} <span>#{$2}</span></h2><div style='display:none' class='content_block' id='content_#{headings}'>"
+        #buttons = "<button class='section_heading show' section_id='#{headings}'>Show</button><button class='section_heading hide' style='display: none' section_id='#{headings}'>Hide</button>"
+        base = "<h2><span class='section_title'>#{$2}</span></h2><div class='content_block' id='content_#{headings}'>"
 
         if headings > 1
           base = "</div>" + base
-- 
1.5.6.3

